### DISCUSSION TITLE
Errors when training LoRA with "formatted dataset" in oobabooga, please.

### DISCUSSION THREAD

**Discussion by hyhahn** [oobabooga/text-generation-webui#3813] [Q&A]
These are the error messages I get in my browser when training the "formatted dataset" example, alpaca_data_cleaned.json, on oobabooga. What could be the problem?
Please resolve errors.

ㅇ Model: TheBloke-LLAMA2-13B-GPTQ
ㅇ GPU : 4090 24GB

<img width="1892" alt="화면 캡처 2023-09-05 165414" src="https://github.com/oobabooga/text-generation-webui/assets/29325885/eb2914c9-b637-4aec-9059-dd52d593fa20">

<img width="1903" alt="화면 캡처 2023-09-05 165434" src="https://github.com/oobabooga/text-generation-webui/assets/29325885/cb5f1df8-2dd8-4772-b907-5f21b2b89aa9">

**Comment by enigmartin** [comment]
This is the same error I'm getting on any of the llama based models. Is there a workaround for this? I've tried quite a few models but none of them seem to work for training.

**Reply by iznanka** [reply]
I am loading the model through transformer with the auto-device flag. Like many people here, the tutorial ends at step 30 with an error.
